keras_pong.py

```diff
@@ -67,32 +63,69 @@
 '''
 model.add(Dense(3, activation=None))
 
-model.compile(loss=keras.losses.categorical_crossentropy,
-              optimizer=keras.optimizers.Adadelta(),
+model.compile(loss=keras.losses.binary_crossentropy,
+              optimizer=keras.optimizers.Adam(),
               metrics=['accuracy'])
 
-
-# model.fit(state, reward, epochs=1, verbose=0)
-
 episodes = 20
-
-future_action = 0
+epsilon = 0.05
+gamma = 0.95
+replayMemory = [None]
+firstAction = true
+batchSize = 10
+possibleActions = [1, 2, 3]
```

Comment:
I think this should be [0, 2, 3] if I remember the enum correctly, but we'll be able to check this.